src/TestCommand.java

```diff
@@ -0,0 +1,20 @@
+import java.util.Scanner;
+
+public class TestCommand implements Command {
```

Comment:
This is like what kind of Test Double?  Have you read 'The Little Mocker'?  I'd read that, then rename this as appropriate.